src/TypeCatalogParser/project.json

```diff
@@ -1,7 +1,8 @@
 {
     "buildOptions": {
         "warningsAsErrors": true,
-        "emitEntryPoint": true
+        "emitEntryPoint": true,
+        "debugType": "portable"
```

Comment:
this is another project that only gets built for .NET Core, so putting `debugType` setting here is good.